src/dwn/interfaces/records.js

```diff
@@ -0,0 +1,103 @@
+import { DwnConstant } from '@tbd54566975/dwn-sdk-js';
+
+import { Interface } from './interface.js';
+import { Record } from '../models/record.js';
+import { dataToBytes } from '../../utils.js';
+
+export class Records extends Interface {
+  constructor(dwn) {
+    super(dwn, 'Records');
+  }
+
+  get create() {
+    return this.write;
+  }
+
+  async createFrom(target, request) {
+    const { author: inheritedAuthor, target: _, ...inheritedProperties } = request.record.toJSON();
```

Comment:
this seems like a super subtle way of not including `target` in `inheritedProperties`

can we do an `delete inheritedProperties.target;` somewhere down below instead?  that seems way more explicit (and gives an opportunity to comment as to why)

Reply:
Addressed in 96c7a8030cff76db65305578bc3beb34ffc6fc51